quickstart/app/Models/Category.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Category extends Model
+{
+    use HasFactory;
+    protected $table = "categories";
+    protected $filltable = [
+        'cat_name',
+        'cat_desc',
+    ];
+
+    /**
+     * Get all of the product for the Category
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\HasMany
+     */
+    public function product(): HasMany
```

Comment:
```suggestion
    public function products(): HasMany
```
hasmany thì để tên số nhiều nhé